components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIAdmin.java

```diff
@@ -603,4 +604,46 @@ List<KeyManagerConfigurationDTO> getGlobalKeyManagerConfigurations(String organi
      * @throws APIManagementException If retrieval fails.
      */
     LLMProvider getLLMProvider(String organization, String llmProviderId) throws APIManagementException;
+
+    /**
+     * 
+     * Retrieves list of organizations available for the given parent organization.
+     * 
+     * @param parentOrgId parent organization id
+     * @param tenantDomain super domain
+     * @return organization list
+     */
+    List<OrganizationDetailsDTO> getOrganizations(String parentOrgId, String tenantDomain) throws APIManagementException;
+
+    /**
+     * Add new organization 
+     * @param orgDto organization details
+     * @return added organization
+     */
+    OrganizationDetailsDTO addOrganization(OrganizationDetailsDTO orgDto) throws APIManagementException;
+
+    /**
+     * Get organization details 
+     * @param organizationId organization id
+     * @param tenantDomain tenant domain
+     * @return
+     * @throws APIManagementException
+     */
+    OrganizationDetailsDTO getOrganizationDetails(String organizationId, String tenantDomain)
+            throws APIManagementException;
+
+    /**
+     * Update organization details 
+     * @param organizationInfoDTO request object
+     * @return updated organization info
+     * @throws APIManagementException
+     */
+    OrganizationDetailsDTO updateOrganization(OrganizationDetailsDTO organizationInfoDTO) throws APIManagementException;
```

Comment:
same goes here

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/12808/commits/7ce47d51d2f253af8332fc5b264f0668ad30b3d8